cmd/release/main.go

```diff
@@ -1,11 +1,21 @@
 package main
 
 import (
+	"bytes"
+	"context"
 	"encoding/json"
+	"fmt"
+	"html/template"
+	"net/url"
 	"os"
 	"os/exec"
+	"regexp"
+	"strconv"
 	"strings"
 
+	masterminds "github.com/Masterminds/semver/v3"
+	sprig "github.com/Masterminds/sprig/v3"
```

Comment:
I don't know the go conventions, but given we have two things coming from "masterminds", should we call this `masterminds_semver` ?

Reply:
it's not a big deal, but `semver` and `sprig` would probably be best, in fact they should be the default for both of these, no? so they may not need the names at all

Reply:
The alias for spring is my Go newbieness.  
I believe masterminds semver is named because another semver library is used below.

Reply:
I have removed the `sprig` label.